### PR TITLE
Make temp_token string optional 

### PR DESCRIPTION
**Pull request by JohnMwashuma**
I have added a check for null in case a temp_token string is not available since it's usually null when absent. This was the reason why the logic of making it optional was not working previously.

### CODE REVIEW

File: src/milia/api/async_export.cljc [hunk 1]
```diff
@@ -19,9 +20,10 @@
    to explicity pass a question-mark or ampersand depending on whether this is
    the first or a subsequenty query parameter."
   [& s]
-  (join (if-let [temp-token (:temp-token *credentials*)]
+  (let [temp-token (:temp-token *credentials*)]
+  (join (if (is-not-null? temp-token)
```
Comment (pld) [inline src/milia/api/async_export.cljc]: this should be indented, since it's w/i the `let`

**Comment by pld** [comment]
👍 cool, so it was coming into through the JS as an actual `null` which the `if-let` said is not a logical false, and would do the wrong thing? Looks like a good fix

**Comment by JohnMwashuma** [comment]
@pld  Yeah that's very correct

File: src/milia/api/async_export.cljc (hunk 1 above)
Comment (JohnMwashuma) [reply src/milia/api/async_export.cljc]: Right nice catch, lemme fix that.

**Pull request merged**